### DISCUSSION TITLE
Issue Regarding polyfills.ts file when I am trying to publish my storybook to Chromatic

### DISCUSSION THREAD

**Discussion by ASH-KING-OP** [storybookjs/storybook#25057] [Documentation]
Hello Guys,

I have developed a Storybook in angular and when I am trying to publish my storybook in chromatic I am getting an error ( Most likely cause is that a Promise polyfill has been loaded after Zone.js (Polyfilling Promise api is not necessary when zone.js is loaded. If you must load one, do so before loading zone.js. )  

Also I am able to build my storybook but facing this error when publishing

I Have Commented my polyfills.ts file but still I am getting an error

My Storybook version is 6.5.8 

Here's The Screenshot of the error 
![Screenshot 2023-11-30 151146](https://github.com/storybookjs/storybook/assets/152498345/63adafb8-72e8-4918-81d8-81f4c402af57)

Please help me guys as I have been facing this error for quite a long time now !!

**Comment by valentinpalkovic** [comment]
Can you provide a minimal reproduction somehow?

**Comment by domyen** [comment]
You'll have to fix this in the built Storybook, reproducible via `build-storybook`, _before_ you publish to Chromatic. 

From a quick look, the error message seems to involve your component's code and not Storybook itself.
